x-pack/legacy/plugins/uptime/public/breadcrumbs.ts

```diff
@@ -7,18 +7,20 @@
 import { i18n } from '@kbn/i18n';
 import { ChromeBreadcrumb } from 'src/core/public';
 
-const makeOverviewBreadcrumb = (search?: string): ChromeBreadcrumb => ({
+const makeOverviewBreadcrumb = (search?: string, basepath?: string): ChromeBreadcrumb => ({
   text: i18n.translate('xpack.uptime.breadcrumbs.overviewBreadcrumbText', {
```

Comment:
Can you please follow camelCase for basePath?

Reply:
Please check https://github.com/elastic/kibana/pull/49428/commits/87c65726969248c76e460e4674b75a7b167c60a2 for the update.